modules/Bio/EnsEMBL/Analysis/Hive/Config/Genome_annotation_conf.pm

```diff
@@ -6725,6 +6759,184 @@ sub pipeline_analyses {
         },
       },
 
+      #################
+      ## PCP
+      #################
+
+      {
+        -logic_name => 'dump_fasta',
+        -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+        -rc_name    => '8GB',
+        -parameters => {
+          cmd => 'mkdir -p ' . $self->o('pcp_output') . ';'.
+          ' perl ' . catfile($self->o('enscode_root_dir'), 'ensembl-analysis', 'scripts', 'pcp', 'get_transcripts.pl').
+          ' -user ' . $self->o('user_r').
+          ' -dna_user ' . $self->o('user_r').
+          ' -dbname ' . $self->o('pcp_name').
+          ' -dna_dbname ' . $self->o('dna_db_name').
+          ' -host ' . $self->o('stringtie_initial_db_server').
+          ' -dna_host ' . $self->o('dna_db_server').
+          ' -port ' . $self->o('stringtie_initial_db_port').
+          ' -dna_port ' . $self->o('dna_db_port').
+          ' > ' . $self->o('pcp_output') . '/' . $self->o('pcp_name') . '.fasta',
+        },
+        -wait_for   => 'load_stringtie_transcripts',
+        -flow_into  => {
+                         1 => ['create_pcp_db'],
+        },
+      },
+
+      {
+        -logic_name => 'create_pcp_db',
+        -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveCreateDatabase',
+        -parameters => {
+          source_db   => $self->o('stringtie_initial_db'),
+          target_db   => $self->o('pcp_db'),
+          create_type => 'copy',
+          force_drop  => 1,
+        },
+        -rc_name    => 'default',
+        -flow_into  => {
+          '1->A' => ['run_cpc2', 'run_rnasamba'],
+          'A->1' => ['impute_coding_genes']
+        },
+      },
+
+      {
+        -logic_name => 'run_cpc2',
+        -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+        -rc_name    => '8GB',
+        -parameters => {
+          cmd => 'sh /hps/nobackup2/production/ensembl/jma/src/python_wrappers/run_CPC2.sh ' .
+          $self->o('pcp_output') . '/' . $self->o('pcp_name') . '.fasta ' .
+          $self->o('pcp_output') . '/' . $self->o('pcp_name') . '_cpc2',
+        },
+      },
+
+      {
+        -logic_name => 'run_rnasamba',
+        -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+        -rc_name    => '10GB',
+        -parameters =>  {
+          cmd => 'sh /hps/nobackup2/production/ensembl/jma/src/python_wrappers/run_RNAsamba.sh ' .
+          $self->o('pcp_output') . '/' . $self->o('pcp_name') . '_RNAsamba.tsv ' .
+          $self->o('pcp_output') . '/' . $self->o('pcp_name').'.fasta ' .
+          $self->o('rna_samba_weights'),
+        },
+        -flow_into =>  {
+          -1 => ['run_rnasamba_50GB'],
+        },
+      },
+
+      {
+        -logic_name => 'run_rnasamba_50GB',
+        -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+        -rc_name    => '50GB',
+        -parameters =>  {
+          cmd => 'sh /hps/nobackup2/production/ensembl/jma/src/python_wrappers/run_RNAsamba.sh ' .
```

Comment:
This should be moved to a more central location and use a variable for the path.

Reply:
Moved to a temp locations. Will arrange a proper central install ahead of integration to dev/hive_master